### ISSUE TITLE
[partially done] Official support for SSR with angular universal

### ISSUE THREAD

**Issue by Ks89** [Ks89/angular-modal-gallery#81]
Scheduled for version 4.0.0

**Comment by Ks89** [comment]
Already supported here https://github.com/Ks89/angular-webpack2-starter-demo-with-angular-modal-gallery

It is working also with 3.x.x, but only on that starter.

I should create a minimal starter and put it into demo's folder.

**Comment by Ks89** [comment]
Server-Side-Rendering (SSR) partially implemented in angular-modal-gallery 5.0.0.
I still have a non blocking issue about Mousetrap.
Also I added an official example with angular-universal.

**Comment by Ks89** [comment]
I should try this solution to provide "window": https://github.com/mlent/vanilla-universal-starter/blob/master/server.js

**Comment by Ks89** [comment]
I decided to release it in version 5.0.0, however angular-universal still doesn't support 'window' object on server side, so I cannot add the full official support.

I you want to try, you'll see that the library itself is working, but your server's log will be polluted with stupid error messages.

When Angular team will add 'window' object on server-side, this library will work automatically without other modifications. For this reason I decided to mark this issue as closed.
